examples/quickstart.php

```diff
@@ -82,7 +84,7 @@
  */
 $eventStore->getActionEventEmitter()->attachListener(
     'commit.post', //Most of the event store methods provide pre and post hooks
-    function (\Prooph\Common\Event\ActionEvent $actionEvent) {
+    $function = function (\Prooph\Common\Event\ActionEvent $actionEvent) use (&$function) {
```

Comment:
Missing return type hint. Should be :\Prooph\Common\Event\DefaultListenerHandler